packages/governance/src/contractGovernor.js

```diff
@@ -0,0 +1,87 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { details as X } from '@agoric/assert';
+
+import { setupGovernance } from './governParam';
+
+/*
+ * ContractManager is an ElectionManager that starts up a contract and hands its
+ * own creator a facet that allows them to call for votes on parameters that
+ * were declared by the contract.
+ *
+ * ContractManager is initialized by a call to startGovernedInstance() on this
+ * contract's creatorFacet. That call initializes the registrar, and identifies
+ * the Installation to be started.
+ *
+ * The governedContract is responsible for supplying getParamMgrAccessor() in
+ * its creatorFacet. getParamMgrAccessor() takes a paramDesc, which identifies
+ * the parameter to be voted on. If the contract has a single paramMgr, the
+ * paramDesc can be just the parameter name. If the contract has more than one,
+ * then it must accept enough details to identify the paramMgr and parameter.
```

Comment:
Trying to identify the paramMgr by the paramName doesn't seem right. What if there are multiple paramNames that are the same? I think it would be better for the governed contract to return an object with string keys and ParamManagerFull values.